test/fixtures/errors/force_colors.snapshot

```diff
@@ -4,10 +4,10 @@ throw new Error('Should include grayed stack trace')
 
 Error: Should include grayed stack trace
     at Object.<anonymous> [90m(/[39mtest*force_colors.js:1:7[90m)[39m
-[90m    at Module._compile (node:internal*modules*cjs*loader:1255:14)[39m
-[90m    at Module._extensions..js (node:internal*modules*cjs*loader:1309:10)[39m
-[90m    at Module.load (node:internal*modules*cjs*loader:1113:32)[39m
-[90m    at Module._load (node:internal*modules*cjs*loader:960:12)[39m
+[90m    at Module._compile (node:internal*modules*cjs*loader:1256:14)[39m
+[90m    at Module._extensions..js (node:internal*modules*cjs*loader:1310:10)[39m
+[90m    at Module.load (node:internal*modules*cjs*loader:1114:32)[39m
+[90m    at Module._load (node:internal*modules*cjs*loader:961:12)[39m
```

Comment:
@MoLow shouldn’t these line/column numbers be obscured so that unrelated changes don’t require updates to snapshots?

Reply:
I agree. Maybe we should replace them with `*`